examples/e2e-tests/kubernetes/release/default/definition.json

```diff
@@ -121,15 +121,6 @@
         "distro": "ubuntu-18.04-gen2",
         "availabilityProfile": "VirtualMachineScaleSets",
         "vnetSubnetId": "/subscriptions/SUB_ID/resourceGroups/RG_NAME/providers/Microsoft.Network/virtualNetworks/VNET_NAME/subnets/SUBNET_NAME"
-      },
-      {
```

Comment:
I don't see any reason to include two Windows pools on the standard PR E2E config

Reply:
me either